readme.md

```diff
@@ -176,12 +223,15 @@ The following are valid conditionals:
 ```
 if 2 > 1 {
 
- print ( "hello" );
+ print ( "Good morning." );
+
+} else if {
```

Comment:
When would this else if ever evaluate true?

Reply:
Sorry, I forgot to put a condition in it.